bluebrain/repo-bluebrain/packages/brayns/package.py

```diff
@@ -7,141 +7,45 @@
 
 
 class Brayns(CMakePackage):
-    """Visualizer for large-scale and interactive ray-tracing of neurons"""
+    """Interactive visualizer for large-scale scientific data
+    """
 
     homepage = "https://github.com/BlueBrain/Brayns"
     git = "https://github.com/BlueBrain/Brayns.git"
     generator = 'Ninja'
-    submodules = True
+    submodules = False
 
-    version('develop', branch='master')
-    version('1.0.1', tag='1.0.1')
-    version('1.1.0', tag='1.1.0')
-    version('2.0.0', tag='2.0.0', submodules=False)
-    version('immersive', branch='videostreaming')
+    version('develop', branch='develop')
+    version('3.0.0', tag='3.0.0')
 
-    variant('assimp', default=True, description='Build with assimp support')
-    variant('ospray', default=True, description='Enable OSPRray engine')
-    variant('deflect', default=True, description='Enable Deflect support')
-    variant('brion', default=False, description='Build CircuitViewer support')
-    variant('net', default=True, description='Enable web interface')
-    variant('opendeck', default=False, description='Enable OpenDeck support')
-    variant('viewer', default=True, description='Build braynsViewer app')
-    variant('optix', default=False, description='Build Optix engine')
-    variant('test', default=False, description='Enable extra tests')
-    variant('doc', default=False, description='Build documentation')
+    variant('blueconfig', default=True, description='BlueConfig support')
 
-    depends_on('cmake@3.1:', type='build')
-    depends_on('doxygen', type='build', when='+doc')
-    depends_on('graphviz', type='build', when='+doc')
-    depends_on('ispc', type='build')
+    depends_on('cmake@3.15:', type='build')
+    depends_on('ispc@1.18.0', type='build')
     depends_on('ninja', type='build')
-
-    depends_on('assimp@5.0.1', when='@1.1.1: +assimp')
-    depends_on('brion', when='+brion')
-    depends_on('ffmpeg@4.2.2', when='+net')
-    depends_on('git', when='@1.1.1:')
-    depends_on('glew', when='+viewer')
+    depends_on('git', type='build')
+
+    depends_on('brion@3.3.9', when='+blueconfig')
+    depends_on('libsonata@0.1.16')
+    depends_on('morphio@3.3.3')
+    depends_on('mvdtool@2.4.2')
+    depends_on('ospray@2.10.0')
+    depends_on('rkcommon@1.10.0')
+    depends_on('spdlog@1.9.2')
+    depends_on('poco@1.11.1')
```

Comment:
It would be great if we could add some version ranges to, i.e., `morphoi`, `libsonata`, and `mvdtool` (or even drop the latter)

Reply:
I have added open range versions for morphio, libsonata, mvdtool and brion